From 4f03a8f6bb426e2a08fb0255519b586e3e46e923 Mon Sep 17 00:00:00 2001
From: Gowtham Anandha Babu <gowtham.anandha.babu@intel.com>
Date: Fri, 9 Aug 2024 11:27:17 +0000
Subject: [PATCH] Add BT HAL aidl service in product packages

As part A15 requirement, the BT HAL has to be upgraded to
AIDL from HIDL.

Removing the hidl and adding aidl implementation for BT HAL.

Tests done:
1. Flash binary
2. Check BT on/off
3. Check BT scan
4. Check BT connect to headset

Tracked-On: OAM-122186
Signed-off-by: Gowtham Anandha Babu <gowtham.anandha.babu@intel.com>
---
 groups/bluetooth/btusb/product.mk        | 3 +--
 groups/device-specific/caas/manifest.xml | 9 ---------
 2 files changed, 1 insertion(+), 11 deletions(-)

diff --git a/groups/bluetooth/btusb/product.mk b/groups/bluetooth/btusb/product.mk
index 8b4d648..e184e6f 100644
--- a/groups/bluetooth/btusb/product.mk
+++ b/groups/bluetooth/btusb/product.mk
@@ -6,8 +6,7 @@ PRODUCT_COPY_FILES += \
 PRODUCT_PROPERTY_OVERRIDES += bluetooth.rfkill=1
 
 PRODUCT_PACKAGES += \
-    android.hardware.bluetooth@1.0-service.vbt \
-    libbt-vendor \
+    android.hardware.bluetooth-service.default.vbt \
     android.hardware.bluetooth.audio-impl \
 
 {{#ivi}}
diff --git a/groups/device-specific/caas/manifest.xml b/groups/device-specific/caas/manifest.xml
index f01e474..6d4745f 100644
--- a/groups/device-specific/caas/manifest.xml
+++ b/groups/device-specific/caas/manifest.xml
@@ -35,15 +35,6 @@
             <instance>default</instance>
         </interface>
     </hal>
-    <hal format="hidl">
-       <name>android.hardware.bluetooth</name>
-       <transport>hwbinder</transport>
-       <version>1.0</version>
-       <interface>
-           <name>IBluetoothHci</name>
-           <instance>default</instance>
-       </interface>
-   </hal>
     <!--hal format="aidl" optional="true">
         <name>android.hardware.bluetooth.audio</name>
         <version>2</version>
-- 
2.45.2

